docs: correct grammatical errors in 1.installation.md (#1710)
nuxt/nuxtjs.org: content/en/docs/1.get-started/1.installation.md

## Using create-nuxt-app

> To get started quickly, you can use the [create-nuxt-app](https://github.com/nuxt/create-nuxt-app).

Reviewer: ```suggestion
To get started quickly, you can use [create-nuxt-app](https://github.com/nuxt/create-nuxt-app).
```

> Make sure you have npx (npx is shipped by default since npm 5.2.0), npm (v6.1 or later) or yarn installed.

Reviewer: ```suggestion
Make sure you have installed yarn, npx (included by default with npm v5.2+) or npm (v6.1+).
```

The application is now running on [http://localhost:3000](http://localhost:3000). Well done!

::alert{type="info"}

> Another way to get started with Nuxt.js is to use [CodeSandbox](https://template.nuxtjs.org), which is a great way for quickly playing around with Nuxt.js and/or sharing your code with other people.

Reviewer: ```suggestion
Another way to get started with Nuxt is to use [CodeSandbox](https://template.nuxtjs.org), which is a great way for quickly playing around with Nuxt and/or sharing your code with other people.
```

```
::

> This command will add `nuxt` as a dependency to your project and include it in your `package.json`. The `node_modules` directory will also be created which is where all your installed packages and dependencies are stored.

Reviewer: I think the original was clearer here.
```suggestion
This command will add `nuxt` as a dependency to your project and add it to your `package.json`. The `node_modules` directory will also be created which is where all your installed packages and dependencies are stored.
```

Author: I think "include" is the more appropriate word but okay.
